Zend/zend_interfaces.stub.php

```diff
@@ -6,44 +6,44 @@ interface Traversable {}
 
 interface IteratorAggregate extends Traversable
 {
-    /** @return Traversable */
-    public function getIterator();
+    /** @tentative-return-type */
+    public function getIterator(): Traversable;
 }
 
 interface Iterator extends Traversable
 {
     /** @return mixed */
-    public function current();
+    public function current(); // TODO Can't add tentative return tyoe due to DirectoryIterator::current()
```

Comment:
There are a few todos like this

Reply:
Add `: mixed` for the DirectoryIterator methods for now? Not wrong, just imprecise...

Reply:
Yeah, I forgot to suggest this workaround, but that's what I'd also prefer.